spyder-custom-layout/spyder_custom_layout/spyder/widgets.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this widget is not necessary because I don't see where you're using the plugin that contains `SpyderCustomLayoutWidget` in the `CustomLayout` class.

If I'm mistaken or I didn't understand things correctly, please let me know how you're using in this example.